packages/blade/src/components/Box/Box.stories.tsx

```diff
@@ -61,7 +63,9 @@ export const Responsive = (args: BoxProps): JSX.Element => {
         minHeight="spacing.10"
         padding="spacing.2"
       >
-        <Text contrast="high">flexDirection property of this layout will change in mobile</Text>
+        <Text contrast="high">
+          Change screen size to see flexDirection switch between row and column
```

Comment:
can make it more like this. and then when we switch the screensize the flexDirection prop will update to column

<img width="809" alt="image" src="https://user-images.githubusercontent.com/11384858/225219056-02159002-97c9-4009-86d9-3db40a4d75f8.png">

Reply:
>  flexDirection prop will update to column

This will add a bit confusing code in that "show code" tab of storybook. But I like this layout. Will make the change.

Reply:
done.

So to add that dynamic flexDirection, we'll have to add
```jsx
<Box display={{ base: 'none', m: 'inline' }}><Text>row</Text></Box>
<Box display={{ base: 'inline', m: 'none' }}><Text>column</Text></Box>
```

Which is a bit confusing code to look at. And kind of unrelated to the example we want to focus on.

Reply:
can you add padding to box 2? OCD 🙈

Reply:
oh lol missed. adding.

Reply:
done